source/scpp/src/scp/BallotProtocol.cpp

```diff
@@ -1564,27 +1561,6 @@ BallotProtocol::hasPreparedBallot(SCPBallot const& ballot,
     return res;
 }
 
-Hash
-BallotProtocol::getCompanionQuorumSetHashFromStatement(SCPStatement const& st)
-{
-    Hash h;
-    switch (st.pledges.type())
-    {
-    case SCP_ST_PREPARE:
-        h = st.pledges.prepare().quorumSetHash;
-        break;
-    case SCP_ST_CONFIRM:
-        h = st.pledges.confirm().quorumSetHash;
-        break;
-    case SCP_ST_EXTERNALIZE:
-        h = st.pledges.externalize().commitQuorumSetHash;
-        break;
```

Comment:
Do you know why Stellar had this difference ?

Reply:
For the line, I don't know the reason which I should analyze more about.

Reply:
The `commitQuorumSetHash` is the same value as `quorumSetHash` of other types of SCPStatement. It's never changed as long as the quorum set remains the same. So it should be removed.

https://github.com/bosagora/agora/blob/bfe95e853eef28f8487d9a2fc8897b86d2852554/source/scpp/src/scp/BallotProtocol.cpp#L560-L576